snapcraft.yaml

```diff
@@ -178,6 +179,8 @@ parts:
       ROOT=https://ftp.mozilla.org/pub/firefox/nightly/latest-mozilla-central
       REVISION=$(wget -O - $ROOT/firefox-$SNAPCRAFT_PROJECT_VERSION.en-US.linux-x86_64.txt | tail -1 | sed -n "s#^$REPO/rev/\(.*\)\$#\1#p")
       hg clone $REPO -u $REVISION .
+      MACH="env -u LD_LIBRARY_PATH -u PYTHONPATH /usr/bin/python3 ./mach"
+      $MACH artifact toolchain --from-build linux64-dump_syms
```

Comment:
I guess that's fine for testing purposes (given that we know nightly builds are done for x64 only), but I'd prefer if there was a way to install the dependencies in an architecture-independent way. Can this artifact be re-created / downloaded from some other place for other architectures?

Reply:
Are non amd64 builds done using cross compilation or natively ?

Reply:
It should not be too complicated to rebuild, but we need to extract the revision as from taskcluster and build it (it is rust), isnt it more problem? https://github.com/mozilla/dump_syms/

Reply:
All builds are done natively on Launchpad.
Building dump_syms natively, provided it isn't too time-consuming, looks cleaner to me.

Reply:
Ok, it seems to be only taking < 2min here, so we can live with that. Works well on my amd64 system